lib/taskana-core/src/main/java/pro/taskana/mappings/WorkbasketMapper.java

```diff
@@ -25,8 +27,8 @@
       value = {
         @Result(property = "id", column = "ID"),
         @Result(property = "key", column = "KEY"),
-        @Result(property = "created", column = "CREATED"),
-        @Result(property = "modified", column = "MODIFIED"),
+        @Result(property = "created", column = "CREATED", jdbcType = JdbcType.TIMESTAMP, javaType = java.time.Instant.class, typeHandler = InstantTypeHandler.class),
```

Comment:
i dont like that. too much risk of forgetting any "timestamp" field. Think about new ones in the future added by someone else ...

Reply:
That's correct. But do you have a better alternative?